sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/ContainerRepositoryClientLiveTests.cs

```diff
@@ -109,7 +113,256 @@ public async Task CanSetRepositoryProperties()
             // Cleanup
             await client.SetPropertiesAsync(originalContentProperties);
         }
+        #endregion
 
+        #region Registry Artifact Tests
+        [RecordedTest]
+        public async Task CanGetMultiArchitectureImageProperties()
+        {
+            // Arrange
+            ContainerRepositoryClient client = CreateClient();
+            string tag = "v1";
+            int helloWorldManifestReferences = 9;
+
+            // Act
+            RegistryArtifactProperties properties = await client.GetRegistryArtifactPropertiesAsync(tag);
+
+            // Assert
+            Assert.Contains(tag, properties.Tags.ToList());
+            Assert.AreEqual(_repositoryName, properties.Repository);
+            Assert.GreaterOrEqual(helloWorldManifestReferences, properties.RegistryArtifacts.Count);
+
+            Assert.IsTrue(properties.RegistryArtifacts.Any(
+                artifact => {
+                    return artifact.CpuArchitecture == "arm64" &&
+                           artifact.OperatingSystem == "linux"; } ));
+
+            Assert.IsTrue(properties.RegistryArtifacts.Any(
+                artifact => {
+                    return artifact.CpuArchitecture == "amd64" &&
+                           artifact.OperatingSystem == "windows";
+                }));
+        }
+
+        [RecordedTest]
+        public async Task CanGetArtifactProperties()
+        {
+            // Arrange
+            ContainerRepositoryClient client = CreateClient();
+            string tag = "v1";
+
+            // Act
+            RegistryArtifactProperties listProperties = await client.GetRegistryArtifactPropertiesAsync(tag);
+            var arm64LinuxImage = listProperties.RegistryArtifacts.Where(
+               artifact =>
+               {
+                   return artifact.CpuArchitecture == "arm64" &&
+                          artifact.OperatingSystem == "linux";
+               }).First();
```

Comment:
nit: You can simplify to remove the `Where`
```suggestion
            var arm64LinuxImage = listProperties.RegistryArtifacts.First(
                artifact =>
                    artifact.CpuArchitecture == "arm64" &&
                    artifact.OperatingSystem == "linux");
```